easybuild/easyconfigs/b/BiG-SCAPE/BiG-SCAPE-1.0.1-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,48 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Tarball'
+
+name = 'BiG-SCAPE'
+version = '1.0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://bigscape-corason.secondarymetabolites.org/index.html'
+description = """BiG-SCAPE and CORASON provide a set of tools to explore the diversity of biosynthetic gene clusters 
+(BGCs) across large numbers of genomes, by constructing BGC sequence similarity networks, grouping BGCs into gene 
+cluster families, and exploring gene cluster diversity linked to enzyme phylogenies."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+source_urls = ["https://git.wageningenur.nl/medema-group/%(namelower)s/-/archive/v%(version)s"]
+sources = ['v%(version)s.tar.gz']
+checksums = ['498aef04db19458c8d92bbb85d62f2f034c71d18f862750d90cf9717d6c0c020']
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('scikit-learn', '0.21.3', versionsuffix),
+    ('networkx', '2.4', versionsuffix),
+    ('HMMER', '3.2.1'),
+    ('FastTree', '2.1.11'),
+]
+
+postinstallcmds = ["cd %(installdir)s && chmod a+x bigscape.py"]
+
+sanity_check_paths = {
+    'files': ['bigscape.py'],
+    'dirs': [],
+}
+
+sanity_check_commands = [
+    'bigscape.py --help',
+]
+
+modextrapaths = {'PATH': ''}
+
+modloadmsg = "%(name)s needs processed Pfam database to work properly.\n"
+modloadmsg += "For this, download the latest 'Pfam-A.hmm.gz' file from the Pfam website"
+modloadmsg += "(ftp://ftp.ebi.ac.uk/pub/databases/Pfam/releases/), "
```

Comment:
actually, can't we make a easyconfig for the database as well, and depend on it?

Reply:
I doubt installing databases where software installed is good practice?

Reply:
hm, ok they were a bit larger than expected